JASP-Engine/JASP/R/selectionmodels.R

```diff
@@ -109,47 +132,36 @@ SelectionModels <- function(jaspResults, dataset, options, state = NULL) {
   x <- trimws(x, which = "both")
   x <- x[x != ""]
   
-  if(anyNA(as.numeric(x)))
+  if (anyNA(as.numeric(x)))
     JASP:::.quitAnalysis(gettext("The p-value cutoffs were set incorectly."))
-  if(length(x) == 0){
+  if (length(x) == 0) {
     JASP:::.quitAnalysis(gettext("At least one p-value cuttoff needs to be set."))
   }
   
   x <- as.numeric(x)
-  if(options[["selection_twosided"]]){
+  if (options[["selection_twosided"]])
     x <- c(x/2, 1-x/2)
-  }
   
-  if(any(x <= 0 | x >= 1)){
+  
+  if (any(x <= 0 | x >= 1))
     JASP:::.quitAnalysis(gettext("All p-value cutoffs needs to be between 0 and 1."))
-  }
+  
   
   x <- c(sort(x), 1)
   
   return(x)
 }
-.SM_pval_get           <- function(dataset, options){
-  # weightfunc uses one-sided p-values as input!
-  if(options[["input_p"]] == ""){
-    pval <- pnorm(
-      dataset[,.v(options[["input_ES"]])] /dataset[,.v(options[["input_SE"]])],
-      lower.tail = options[["effect_direction"]] == "negative") 
-  }else{
-    pval <- dataset[,.v(options[["input_p"]])]
-  }
-  return(pval)
-}
-.SM_autoreduce         <- function(steps, pval){
-
+.smAutoreduceCutoffs       <- function(steps, pval) {
```

Comment:
Why the auto reduce? Why isn't it just a reduce?

Reply:
changed

Reply:
the option in UI is called "Automatically reduce p-value intervals" though it was closer to the original naming. It was renamed by then, so I changed it to `smJoinCutoffs`